src/main/java/io/github/hypixel_api_wrapper/wrapper/util/GuildLevelingUtil.java

```diff
@@ -0,0 +1,159 @@
+package io.github.hypixel_api_wrapper.wrapper.util;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+// source code: https://github.com/HypixelDev/PublicAPI/blob/master/hypixel-api-core/src/main/java/net/hypixel/api/util/IGuildLeveling.java
+public class GuildLevelingUtil {
+
+
+    /**
+     * An unmodifiable list containing the experience needed to get from each level to the next. For
+     * example, the value at index 0 is the amount of guild experience needed to progress from guild
+     * level 0 to 1. The value at index 7 is the guild exp needed to progress from level 7 -> 8.
+     * Etc.
+     * <p>
+     * The last element in this list is the value used for any levels beyond the size of this list.
+     * For example, if this list has 15 values, then the last value is used for levels 14 -> 15 and
+     * any levels after that.
+     */
+    static List<Integer> EXP_NEEDED = Collections.unmodifiableList(Arrays.asList(
```

Comment:
This can be private + converted to a plain array since it's not modified within the class anyways

Reply:
Sounds good